.github/workflows/canary-integration-test.yml

```diff
@@ -97,6 +105,17 @@ jobs:
         kubectl -n rook-ceph exec $toolbox -- rbd namespace create replicapool/radosNamespace
         kubectl -n rook-ceph exec $toolbox -- python3 /etc/ceph/create-external-cluster-resources.py --cephfs-filesystem-name myfs --rbd-data-pool-name replicapool --rados-namespace radosNamespace --cluster-name rookStorage --restricted-auth-permission true
 
+    - name: test the upgrade flag for restricted auth user
```

Comment:
Also, can we move the unit test run to the end after all the flag testing?

Reply:
okay